service_implementation/advanced_plm.md

```diff
@@ -1 +1,455 @@
-# Beyond just files: Using PLM for versioning and metadata
\ No newline at end of file
+# Beyond just files: Using PLM for versioning and metadata
+This document describes how to use PLM services from _inside_ a service, i.e., by calling the PLM API. The text includes  example code.
+There are two types of PLM services for use in CAxMan; both are described below:
+1) those that are integrated in the GSS framework;
+2) the PLM Server native web-services.
+
+## GSS
+
+The PLM Server has its own implementation of the GSS services. GSS offers SOAP and REST services:

```

Comment:
I find this sentence and the general usage of the term "GSS" misleading.
GSS is "Generic Storage Services" and the PLM implementation is certainly _not_ generic. In my view, there is exactly one implementation of GSS (the one from SINTEF) which provides a single generic API to access different storage services. Then there are GSS backends (which in fact are part of the GSS service itself), which provide a translation layer from the generic API to a storage-specific API.
Jotne PLM is a storage service/provider, and what is here called the "PLM implementation of the GSS services" is simply the storage-specific API we call from the PLM backend in GSS.
This confused me heavily at our first meeting at Jotne when we talked about the extensions to be made to the PLM API, and I think it will confuse partners who read this article.

I suggest to use "PLM API" or similar instead of GSS.